autoPyTorch/api/base_task.py

```diff
@@ -1344,7 +1346,7 @@ def refit(
                 dataset_properties=dataset_properties,
                 dataset=dataset,
                 split_id=split_id)
-            fit_and_suppress_warnings(self._logger, model, X, y=None)
+            fit_pipeline(self._logger, model, X, y=None)
```

Comment:
I think the previous name emphasised the fact that we are suppressing warnings, otherwise, we could have just used `model.fit(X, y)`. So could you change this name back to what it was? Or I dont mind `fit_pipeline_supress_warnings`

Reply:
This is another question that is mentioned in the other comment.

This is maybe a stupid question, but why do we need to emphasize `suppress warnings`?
I understand it if we squeeze errors during the fit, but warnings will not be critical for running a script and we write them in logging anyways.
So it is more like `fit_pipeline_with_logging_warnings`?